.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 install:
   - pip install nose coverage argparse python-dateutil docutils pyparsing
   - pip install flake8 flake8-blind-except flake8-builtins flake8-class-newline flake8-comprehensions flake8-deprecated
+  - pip install flake8-docstrings
```

Comment:
Please insert this package into the previous line which already installs all flake8 plugins.

Reply:
Done.  Please note that I plan to add 2 more packages, and the line will be even longer.